dns/bind/src/opnsense/scripts/OPNsense/Bind/dnsbl.sh

```diff
@@ -137,7 +138,14 @@ install() {
 	rm -rf ${WORKDIR}
 }
 
-for CAT in $(echo ${1} | tr ',' ' '); do
+if [ $# -eq 0 ] && [ -s "$DESTDIR/dnsbl.list" ]; then
```

Comment:
Is `-a` not working here?

Reply:
this is what Fabian meant

```suggestion
if [ $# -eq 0 -a -s "$DESTDIR/dnsbl.list" ]; then
```

Reply:
oh, sure - why not.
I'd still change echo to printf, though.